INSTALL.md

```diff
@@ -70,6 +70,19 @@ settings that normally require modification.
 Start the server with `sudo systemctl start owntone` and check that it is
 running with `sudo systemctl status owntone`.
 
+To run multiple instances of owntone on a server, you should copy
```

Comment:
I don't think this belongs in the "Quick version", or even in the INSTALL file. This is about configuration, and running the daemon, so instead it should go into the README.

Reply:
Well, INSTALL was the only place that referenced `systemctl` anywhere.  But I moved this paragraph to README (I created a new section).